### PR TITLE
get and show data list borrower

### PR DESCRIPTION
**Pull request by at-trampham**
## Description
Can see list borrower, I want to see Employe code, Name, Email, Book (name), From date, End date(Get and show data)
## Reference Requirements
- Product Backlog ID: PBI-3
- Task ID: #3.2

## Reference Isuees
Non

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
select * from `users` where `id` = '11' and `users`.`deleted_at` is null limit 1
image
select count(*) as aggregate from `borrows` inner join `users` on `users`.`id` = `borrows`.`user_id` inner join `books` on `books`.`id` = `borrows`.`book_id` where `borrows`.`status` = '0'
image
select `users`.`employ_code`, `users`.`name`, `users`.`email`, `books`.`title`, `borrows`.`from_date`, `borrows`.`to_date`, `borrows`.`id` from `borrows` inner join `users` on `users`.`id` = `borrows`.`user_id` inner join `books` on `books`.`id` = `borrows`.`book_id` where `borrows`.`status` = '0' limit 10 offset 0
image

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/backend/borrows/index.blade.php [hunk 1]
```diff
@@ -36,32 +35,23 @@
...
                     <ul class="pagination">
-                      <li><a href="#">&laquo;</a></li>
-                      <li><a href="#">1</a></li>
-                      <li><a href="#">2</a></li>
-                      <li><a href="#">3</a></li>
-                      <li><a href="#">&raquo;</a></li>
+                      {{ $borrows->links() }}
                     </ul>
                 </nav>
               </div>
```
Comment (at-duongtruong) [inline resources/views/backend/borrows/index.blade.php]: GLTM

File: app/Http/Controllers/Admin/BorrowController.php [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 use Illuminate\Http\Request;
 use App\Http\Controllers\Controller;
+use App\Model\Borrow;
+use App\Model\User;
+use App\Model\Book;
+use DB;
```
Comment (at-tuanphan) [inline app/Http/Controllers/Admin/BorrowController.php]: `DB, User,Book` không dùng thì remove đi em

File: resources/views/backend/borrows/index.blade.php [hunk 3]
```diff
@@ -36,32 +35,23 @@
...
-                    <td class="text-center">28-01-2018</td>
-                  </tr>
-                  <tr>
-                    <td class="text-center">ATI2019</td>
-                    <td>Nhan Nguyen T.</td>
-                    <td>nhan.nguyen@asiantech.vn</td>
-                    <td>HTML5</td>
-                    <td class="text-center">21-01-2018</td>
-                    <td class="text-center">27-01-2018</td>
+                  @foreach ($borrows as $index => $borrow)
```
Comment (at-tuanphan) [inline resources/views/backend/borrows/index.blade.php]: $index có dùng không em?

**Review by ghost — COMMENTED** [review]
please check my comments

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**